impl/src/templates/crate.BUILD.template

```diff
@@ -4,42 +4,45 @@ cargo-raze crate build file.
 
 DO NOT EDIT! Replaced on runs of cargo-raze
 """
-package(default_visibility = [
-  # Public for visibility by "@raze__crate__version//" targets.
-  #
-  # Prefer access through "{{workspace.workspace_path}}", which limits external
-  # visibility to explicit Cargo.toml dependencies.
-  "//visibility:public",
-])
-
-licenses([
-  "{{crate.license.rating}}", # {{crate.license.name}}
-])
 
+# buildifier: disable=load
```

Comment:
For my information, what is wrong here that we need this disable?

Reply:
We can't guarantee that all these macros are used in a generated file. If we don't always disable this warning we'd currently get a wall of the following:
```command
buildifier -lint=warn -r examples
```
```output
examples/vendored/complicated_cargo_library/cargo/vendor/MacTypes-sys-2.1.0/BUILD:10: load: Loaded symbol "rust_binary" is unused. Please remove it.
To disable the warning, add '@unused' in a comment. (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#load)
examples/vendored/complicated_cargo_library/cargo/vendor/MacTypes-sys-2.1.0/BUILD:12: load: Loaded symbol "rust_test" is unused. Please remove it.
To disable the warning, add '@unused' in a comment. (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#load)
examples/vendored/complicated_cargo_library/cargo/vendor/aho-corasick-0.6.10/BUILD:12: load: Loaded symbol "rust_test" is unused. Please remove it.
To disable the warning, add '@unused' in a comment. (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#load)
examples/vendored/complicated_cargo_library/cargo/vendor/arrayvec-0.3.25/BUILD:10: load: Loaded symbol "rust_binary" is unused. Please remove it.
To disable the warning, add '@unused' in a comment. (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#load)
examples/vendored/complicated_cargo_library/cargo/vendor/arrayvec-0.3.25/BUILD:12: load: Loaded symbol "rust_test" is unused. Please remove it.
To disable the warning, add '@unused' in a comment. (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#load)
examples/vendored/complicated_cargo_library/cargo/vendor/arrayvec-0.4.10/BUILD:10: load: Loaded symbol "rust_binary" is unused. Please remove it.
To disable the warning, add '@unused' in a comment. (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#load)
```